wcbench.sh

```diff
@@ -572,10 +588,23 @@ install_opendaylight()
 
     # Install required packages
     echo "Installing OpenDaylight dependencies"
-    if "$VERBOSE" = true; then
-        sudo yum install -y java-1.7.0-openjdk unzip wget
+
+    if ["$OS" == 'fedora']; then    #Please check if "echo $OSTYPE" returns "fedora". If not replace "fedora" here with the returned name
+        if "$VERBOSE" = true; then
+            sudo yum install -y java-1.7.0-openjdk unzip wget
+        else
+            sudo yum install -y java-1.7.0-openjdk unzip wget &> /dev/null
+        fi
+
+    elif [ "$OS" == 'linux-gnu']; then
```

Comment:
This is what Fedora OSs return, need a better check or to manually set the flag.